server/lib/statsd-collector.js

```diff
@@ -157,16 +166,33 @@ function sendNavigationTiming(context, navigationTiming, tags) {
       if (isEventOffsetValid(offset)) {
         var type = 'navigationTiming.' + key;
 
-        if (isEventOffsetInRange(offset, 0, NAVIGATION_TIMING_MAX_OFFSET)) {
+        if (isInRange(offset, 0, NAVIGATION_TIMING_MAX_OFFSET)) {
           context.timing(type, offset, tags);
         } else {
-          logOutOfRangeOffset(type, offset, 0, NAVIGATION_TIMING_MAX_OFFSET);
+          logOutOfRange(type, offset, 0, NAVIGATION_TIMING_MAX_OFFSET);
         }
       }
     }
   }
 }
 
+function sendNumStoredAccounts(context, numStoredAccounts, tags) {
+  if (isCountValid(numStoredAccounts)) {
+    const type = 'num_stored_accounts';
+    if (isInRange(numStoredAccounts, 0, STORED_ACCOUNTS_MAX_COUNT)) {
+      const reportedNumStoredAccounts =
+        Math.min(numStoredAccounts, STORED_ACCOUNTS_MAX_REPORTED_COUNT);
+      // DataDog distribution graphs allow a particular event to be segmented
+      // by tag. Convert the count to a tag so we can see which tag has
+      // the most values.
+      context.increment(
+        type, ['has:' + reportedNumStoredAccounts].concat(tags));
```

Comment:
> Will there be enough other context on the events to remember what that's about when looking at the graphs 6 months from now

This was my worry as writing the PR. I have no existential problems having a tag and event name being almost identical: event name: `num_stored_accounts`, tag name: `num_stored_accounts:0`, `num_stored_accounts:1` or `num_stored_accounts:2`

@vladikoff - seem reasonable?